molgenis-data-security/src/test/java/org/molgenis/data/security/auth/TokenTest.java

```diff
@@ -1,43 +1,20 @@
 package org.molgenis.data.security.auth;
 
-import static java.time.Instant.now;
-import static java.time.temporal.ChronoUnit.MINUTES;
-import static java.time.temporal.ChronoUnit.SECONDS;
-import static org.testng.Assert.assertFalse;
-import static org.testng.Assert.assertTrue;
-
-import org.molgenis.data.AbstractMolgenisSpringTest;
+import org.molgenis.data.meta.AbstractSystemEntityTest;
+import org.molgenis.data.security.config.SecurityTestConfig;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.test.context.ContextConfiguration;
-import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
-@ContextConfiguration(
-    classes = {TokenFactory.class, TokenMetaData.class, SecurityPackage.class, UserMetadata.class})
-public class TokenTest extends AbstractMolgenisSpringTest {
-  @Autowired private TokenFactory tokenFactory;
-
-  private Token token;
+@ContextConfiguration(classes = {SecurityTestConfig.class})
+public class TokenTest extends AbstractSystemEntityTest {
 
-  @BeforeMethod
-  public void setup() {
-    token = tokenFactory.create();
-  }
-
-  @Test
-  public void testIsExpiredNoExpirationDate() throws Exception {
-    assertFalse(token.isExpired());
-  }
-
-  @Test
-  public void testIsExpiredExpirationDateInFuture() throws Exception {
-    token.setExpirationDate(now().plus(1, MINUTES));
-    assertFalse(token.isExpired());
```

Comment:
Wouldn't we like to keep the specific tests? this behaviour is not tested in the new test framework

Reply:
Sharp! Not sure what I was thinking when I removed the original file...